code/math/vecmat.cpp

```diff
@@ -1397,7 +1397,7 @@ void compute_point_on_plane(vec3d *q, const plane *planep, const vec3d *p)
 
 	vm_vec_scale_add(q, p, &normal, -k);
 
-	tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
+	// tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
```

Comment:
Would probably be preferable to delete if it is truly unnecessary.  Also would be nicer still to prove that this is vestigial, and identify when the code that used to use this (if it ever was) was removed.